osgp/platform/osgp-secret-management/src/main/resources/osgp-secret-management.properties

```diff
@@ -28,3 +28,6 @@ spring.jpa.hibernate.naming.implicit-strategy=org.springframework.boot.orm.jpa.h
 
 #Flyway configuration
 spring.flyway.table=schema_version
+
+
```

Comment:
Two empty lines seems like a bit much.
Would it make sense to put this new setting related to the new key closer to the other settings involving keys and secret management in this file as these seem conceptually related, and I think it is more convenient to order things conceptually than from top to bottom by time of inclusion.

Reply:
The **max.duration.for.new.key.to.be.activated** property is a relict of the implementation in secret-management to prevent simultaneously execution of multiple key changing processes. A mechanism to prevent this is now implemented in dlms-adapter. Therefor the property is removed.